src/theme/DocItem/Footer/styles.module.css

```diff
@@ -0,0 +1,93 @@
+.lastUpdated {
```

Comment:
Could you pick 1 naming style for all CSS classes? Either kebab-case or camelCase. Currently this file uses both camelCase and snake_case.